src/cladetime/util/config.py

```diff
@@ -1,77 +1,34 @@
-# mypy: ignore-errors
-
 from dataclasses import InitVar, asdict, dataclass, field
 from datetime import datetime, timezone
+from pathlib import Path
 from pprint import pprint
 
-from cloudpathlib import AnyPath
-
 
 @dataclass
 class Config:
-    sequence_released_date: InitVar[datetime]
-    tree_as_of_date: InitVar[datetime]
-    data_path_root: InitVar[str] = AnyPath(".")
-    sequence_released_since_date: str = None
-    reference_tree_date: str = None
+    data_path_root: InitVar[str] = Path(".")
     now = datetime.now()
     run_time = now.strftime("%Y%m%dT%H%M%S")
-    ncbi_base_url: str = "https://api.ncbi.nlm.nih.gov/datasets/v2alpha/virus/genome/download"
-    ncbi_package_name: str = "ncbi.zip"
-    ncbi_sequence_file: AnyPath = None
-    ncbi_sequence_metadata_file: AnyPath = None
 
-    # Nextstrain sequence data files in their current format is published back to 2023-05-01
+    # Nextstrain sequence data files in their current format are published back to 2023-05-01
     nextstrain_min_seq_date: datetime = datetime(2023, 5, 1).replace(tzinfo=timezone.utc)
-
     # Nextstrain ncov pipeline metadata began publishing on 2024-08-01
     nextstrain_min_ncov_metadata_date: datetime = datetime(2024, 8, 1, 1, 26, 29, tzinfo=timezone.utc)
-
     nextstrain_ncov_bucket = "nextstrain-data"
     nextstrain_ncov_metadata_key = "files/ncov/open/metadata_version.json"
     nextstrain_genome_metadata_key = "files/ncov/open/metadata.tsv.zst"
     nextstrain_genome_sequence_key = "files/ncov/open/sequences.fasta.zst"
     nextclade_base_url: str = "https://nextstrain.org/nextclade/sars-cov-2"
-    reference_tree_file: AnyPath = None
-    root_sequence_file: AnyPath = None
-    assignment_no_metadata_file: AnyPath = None
-    assignment_file: AnyPath = None
     assignment_file_columns: list[str] = field(default_factory=list)
 
     def __post_init__(
         self,
-        sequence_released_date: datetime,
-        tree_as_of_date: datetime,
         data_path_root: str | None,
     ):
         if data_path_root:
-            self.data_path = AnyPath(data_path_root)
+            self.data_path = Path(data_path_root)
```

Comment:
AnyPath wasn't really helping us but _was_ responsible for a lot of annoying type errors, so switched back to good old Path